plugins/net_plugin/net_plugin.cpp

```diff
@@ -1496,14 +1451,14 @@ namespace eosio {
       ds.write( reinterpret_cast<char*>(&packsiz), sizeof(packsiz) );
       fc::raw::pack( ds, msg );
 
-      uint16_t bn = static_cast<uint16_t>(txn.ref_block_num);
+      uint32_t bn = static_cast<uint32_t>(txn.ref_block_num);
```

Comment:
Does this cast do what is intended? Is the 16 bit block number later treated like a full block number?